partials/tabs/home.html

```diff
@@ -33,8 +33,14 @@ <h5 class="light">simply install coala</h5>
                     </p>
                 </div>
                 <div class="col m6 s12">
+                    <p>In order to use coala, you need Python installed. In order to do so, you should install Python >= 3.4
```

Comment:
I recommend merging these lines and condensing the sentences a bit for brevity and adding a link to pip installation guide as well, therefore making it something like:
> In order to use coala, you should have [Python >= 3.4](https://www.python.org/downloads/) and [pip >= 8](https://pip.pypa.io/en/stable/installing/) installed.